invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchResultItem.js

```diff
@@ -12,13 +12,25 @@ import { Table } from "semantic-ui-react";
 import isEmpty from "lodash/isEmpty";
 import { Actions } from "../actions/Actions";
 import { withState } from "react-searchkit";
+import DateFormatter from "../components/DateFormatter";
 
 class SearchResultItemComponent extends Component {
   refreshAfterAction = () => {
     const { updateQueryState, currentQueryState } = this.props;
     updateQueryState(currentQueryState);
   };
 
+  displayValue = (property) => {
```

Comment:
should this component be aware of how to format values? what is the components' responsibility?